lxl-web/src/lib/utils/codemirror/getEditedParts.ts

```diff
@@ -53,4 +62,25 @@ function getEditedParts({ value, cursor }: { value: string; cursor: number }): {
 	}
 }
 
+/**
+ * Returns the index positions of blank spaces outside of quotes in a string
+ */
+function blankSpacesOutsideOfQuotes(str: string) {
```

Comment:
I'm thinking we should maybe look for all kinds of whitespaces (`/\s`)? instead of only blankspaces outside of the quotes (multiline editing isn't implemented now but is quite probable in the future).